Include/internal/pycore_frame.h

```diff
@@ -159,13 +159,6 @@ static inline void _PyFrame_Copy(_PyInterpreterFrame *src, _PyInterpreterFrame *
     // Don't leave a dangling pointer to the old frame when creating generators
     // and coroutines:
     dest->previous = NULL;
-
-#ifdef Py_GIL_DISABLED
```

Comment:
I think this isn't the only place we do this. There's `_PyFrame_Initialize` (and maybe the shim frame stuff).

Reply:
Thanks, I removed the extra initialization from `_PyFrame_Initialize` and `_PyFrame_PushTrampolineUnchecked`.